main.go

```diff
@@ -141,15 +141,14 @@ func handleConflicts(conflicts map[int]lr1Items.RowConflicts, numSets int, cfg c
 	if len(conflicts) <= 0 {
 		return
 	}
-	switch {
-	case !cfg.AutoResolveLRConf():
-		fmt.Printf("Error: %d LR-1 conflicts\n", len(conflicts))
+	if cfg.Verbose() {
 		io.WriteFileString(path.Join(cfg.OutDir(), "LR1_conflicts.txt"), conflictString(conflicts, numSets, prods))
+	}
+	if !cfg.AutoResolveLRConf() {
+		fmt.Printf("Error: %d LR-1 conflicts \n", len(conflicts))
 		os.Exit(1)
-	case cfg.Verbose():
-		fmt.Printf("%d LR-1 conflicts \n", len(conflicts))
-		io.WriteFileString(path.Join(cfg.OutDir(), "LR1_conflicts.txt"), conflictString(conflicts, numSets, prods))
 	}
+	fmt.Printf("%d LR-1 conflicts \n", len(conflicts))
```

Comment:
Move this line to before the `io.WriteFileString` line, as then you get some user output before the call to `conflictString` which could take a long time to compute.

Reply:
Done!